src/UI/Components/TreeTable/TreeRow/TreeRowView.tsx

```diff
@@ -170,11 +176,17 @@ export const TreeRowView: React.FC<RowProps> = ({
               />
             )
           )}
-        </Tr>
+        </StyledTr>
       );
   }
 };
 
 const Spacer = styled.span`
   padding-left: 10px;
 `;
+
+const StyledTr = styled(Tr)`
+  > * {
+    font-size: 16px !important;
```

Comment:
isn't it something we could change in the global css file ?

Reply:
I tried `--pf-c-table--cell--FontSize: 16px;` without `> * {}` which didn't work, but I went back to it and tried inside that selector and it's working :D

Reply:
I meant in the global styling css file, but maybe it gets overriden ? I don't know.